package.json

```diff
@@ -11,11 +11,8 @@
   "author": "Adam Miller <ammiller@linkedin.com>",
   "license": "BSD-2-Clause",
   "scripts": {
-    "start": "yarn run watch",
-    "build": "tsc -p tsconfig.json",
-    "watch": "watch 'yarn run test' './src' './test' --wait=3",
-    "pretest": "yarn run build",
-    "test": "mocha --opts test/mocha.opts dist/test"
+    "build": "yarn lerna run build",
+    "test": "yarn lerna run test"
```

Comment:
you don't need `yarn` in front of these scripts. You'll already be in a yarn environment and so the `lerna` executable will be on the PATH.

Reply:
I think adding `"prepare": "lerna run build"` to the scripts will be nice. it will cause a bare `yarn` to set up all the projects after installing dependencies for the multirepo.